Makefile

```diff
@@ -1,10 +1,16 @@
 all: test build
 
+ui: ui/index.html ui/main.js
+	yarn build
+
 test:
 	go test ./..
 
-build:
-	CGO_ENABLED=0 go build -o ./bin/webui ./cmd/webui
+assets: ui
+	go run -tags=dev pkg/assets/generate.go
```

Comment:
Minor nit: `build` or `generate` might be a more accurate tag to use here -- `dev` is usually reserved for things that you only run locally (i.e., while trying out new code).

Reply:
Fixed